e2e/test/scenarios/question-reproductions/reproductions-1.cy.spec.js

```diff
@@ -312,9 +312,9 @@ describe("postgres > user > query", { tags: "@external" }, () => {
   });
 });
 
-const PG_DB_NAME = "QA Postgres12";
```

Comment:
If we're going to keep all reproductions together, we should be careful about the variable scope.